frontend/src/modules/lf/segments/pages/lf-projects-page.vue

```diff
@@ -2,32 +2,37 @@
   <app-page-wrapper>
     <router-link
       v-if="hasPermission(LfPermission.projectGroupEdit)"
-      class="text-gray-600 btn-link--md btn-link--secondary p-0 inline-flex items-center pb-6"
       :to="{
         name: 'adminPanel',
         query: {
           activeTab: 'project-groups',
         },
       }"
     >
-      <lf-icon name="chevron-left" :size="12" class="mr-2" />
-      <span>Project groups</span>
+      <lf-button type="secondary-ghost" class="mb-6">
+        <lf-icon name="angle-left" type="regular" />
+        <span>Project groups</span>
+      </lf-button>
     </router-link>
-    <div class="text-sm text-primary-500 pb-2">
-      {{ projectGroupForm.name }}
-    </div>
 
-    <div class="w-full flex items-center justify-between mb-6">
-      <h4 class="text-gray-900">
-        Manage projects
-      </h4>
-      <el-button
+    <div class="w-full flex items-center justify-between mb-6 pt-7 border-t border-gray-100">
+      <div class="flex items-center gap-4 whitespace-nowrap">
+        <h4 class="text-gray-900 flex items-center">
+          {{ projectGroupForm.name }}
+        </h4>
+        <app-lf-status-pill :status="projectGroupForm.status" />
```

Comment:
On the designs this one doesn't have a fixed width.
Can you align with Nuno if it's a design inconsistency or not?
If it's on purpose than the component should be more flexible, and we should allow to specify width behaviour

Reply:
Are you referring to the status pill? I didn't put a fix width to this.

Reply:
Yeah. I believe it looks good now